src/Surfnet/StepupGateway/GatewayBundle/Tests/Service/GatewayLoginServiceTest.php

```diff
@@ -169,18 +180,131 @@ public function it_should_on_sso_return_a_proxy_request()
      */
     public function it_should_handle_consume_assertion(AuthnRequest $proxyRequest) {
```

Comment:
Again I'm no big fan of this method name. I'd prefer something like `it_should_handle_assertion_consumption` or `it_should_consume_the_assertion`